tests/include/verifier.hpp

```diff
@@ -108,23 +108,15 @@ namespace gridtools {
             return out;
         }
 
-        template <class Expected, class DataStore, class Halos, class EqualTo = default_equal_to>
-        std::enable_if_t<storage::is_data_store<DataStore>::value &&
-                             is_view_compatible<Expected, DataStore::ndims>::value,
-            bool>
-        verify_data_store(Expected const &expected,
-            std::shared_ptr<DataStore> const &actual,
-            Halos const &halos,
+        template <class Expected, class Actual, size_t NDims, class EqualTo = default_equal_to>
+        bool verify_generic(Expected const &expected,
```

Comment:
I called this `verify_generic` now. Also I don't need it anymore in this PR (I found another way to express the check in fn_unstructured_nabla), but I think it's a worthy cleanup. If we need it at some point, we can just export `verify_generic`.